src/common/common.c

```diff
@@ -32,25 +32,3 @@ OQS_API void OQS_MEM_secure_free(void *ptr, size_t len) {
 OQS_API void OQS_MEM_insecure_free(void *ptr) {
 	free(ptr); // IGNORE free-check
 }
-
-/* Displays hexadecimal strings */
-void OQS_print_hex_string(const char *label, const uint8_t *str, size_t len) {
```

Comment:
OQS_print_hex_string() is also duplicated:
```
./src/kem/kat_kem.c:12:void OQS_print_hex_string(const char *label, const uint8_t *str, size_t len) {
./src/kem/test_kem.c:9:void OQS_print_hex_string(const char *label, const uint8_t *str, size_t len) {
```
Should test _kem just include from a common one?

Reply:
We had a common one before and I moved it out of the common files intentionally.  Otherwise it would be part of our public API, and that seemed unnecessary.